pulp_smash/tests/ostree/api_v2/test_copy.py

```diff
@@ -0,0 +1,73 @@
+# coding=utf-8
+"""Copy content between OSTree repositories with a filter."""
+import unittest
+from urllib.parse import urljoin
+
+from pulp_smash import api, config, utils
+from pulp_smash.constants import OSTREE_BRANCHES, OSTREE_FEED, REPOSITORY_PATH
+from pulp_smash.tests.ostree.utils import gen_distributor, gen_repo
+from pulp_smash.tests.ostree.utils import set_up_module as setUpModule  # noqa pylint:disable=unused-import
+
+
+class CopyTestCase(unittest.TestCase):
+    """Copy content between OSTree repositories with a filter."""
+
+    def test_all(self):
+        """Copy content between OSTree repositories with a filter..
+
+        Verify that:
+
+        Filters can be used to restrict which units
+        are copied between repositories.
+        """
+        repos = []  # list to store data related to created repos
+        cfg = config.get_config()
+        client = api.Client(cfg, api.json_handler)
+
+        # Create the 1st repository with at least 2 content units
+        body = gen_repo()
+        body['importer_config']['feed'] = OSTREE_FEED
+        body['importer_config']['branches'] = OSTREE_BRANCHES
+        body['distributors'].append(gen_distributor())
+        repo = client.post(REPOSITORY_PATH, body)
+        self.addCleanup(client.delete, repo['_href'])
+        repos.append(repo)
+
+        # Sync the 1st repository
+        utils.sync_repo(cfg, repo)
+        search_repo = utils.search_units(cfg, repos[0])
+        first_unit_first_repo = search_repo[0]['metadata']['_id']
+        second_unit_first_repo = search_repo[1]['metadata']['_id']
+
+        # Create the 2nd repository
+        body = gen_repo()
+        repo = client.post(REPOSITORY_PATH, body)
+        self.addCleanup(client.delete, repo['_href'])
+        repos.append(repo)
```

Comment:
How about one variable instead of two? There's a cognitive overhead associated with having many variables in a given scope.

```python
repos.append(client.post(REPOSITORY_PATH, body))
self.addCleanup(client.delete, repos[1]['_href'])
```